src/com/goide/inspections/GoMultiplePackagesInspection.java

```diff
@@ -32,7 +32,7 @@
 public class GoMultiplePackagesInspection extends GoInspectionBase {
   @Override
   protected void checkFile(@NotNull GoFile file, @NotNull ProblemsHolder problemsHolder) {
-    if (((ScratchFileType)ScratchFileType.INSTANCE).isMyFileType(file.getVirtualFile())) return;
+    if (file.isScratchFile()) return;
```

Comment:
Sorry, I had the same check in `GoAutoImportCompletionContributor` until realised that check is not needed.
I will move this change to another pull request.